projects/GKv2/GKPedigreeImporterPlugin/GKPedigreeImporterPlugin.enu

```diff
@@ -27,7 +27,7 @@ Next >
 Close
 Special settings of personal lines format
 1) "Number. Name (*birth +death)" - the dates in brackets
-Analysis
+Analyzing

```

Comment:
This is on other hand is a progress label, so continuous tense would be more appropriate.